src/pages/index.js

```diff
@@ -106,44 +66,108 @@ const settings = {
   errorClass: "modal__error_visible",
 };
 
+/// API classes ///
+
+const apiInstance = new Api({
+  baseUrl: "https://around-api.en.tripleten-services.com/v1",
+  headers: "ca29b9d7-c085-4996-bfd4-aa6f252bbde8",
+  contentType: "application/json",
+});
+
+apiInstance
+  .getInitialCards() // metodo de api class para obtener las cards
+  .then((cards) => {
+    // puse cards, pero tambien se usa "res"de respuesta
+    console.log(cards);
+    cardSection.renderItems(cards);
+  })
+  .catch((err) => console.error("I got an error:", err.message));
+
+apiInstance // llamo al metodo de la api class para actualizar el usuario
+  .getUserInfo()
+  .then((userUpdate) => {
+    console.log(userUpdate);
+    newUserInfo.setUserInfo(userUpdate.name, userUpdate.about);
+  })
+  .catch((err) => console.log.error("I got an error:", err.message));
+
+function handleDelete(card) {
+  // funcion que integra el popup con el api request class, y la uso en la card class
+  deletePopup.open(); // abre el popup
+  deletePopup.setSubmitAction(() => {
+    // llamo a la funcion, esta funcion actua como conexion entre popup, el request y la card en si
+    const cardId = card.getCardId();
+    apiInstance
```

Comment:
Needs correction: don't forget to notify the user about the loading process by changing the confirm button text